src/services/Store/store/account.slice.ts

```diff
@@ -275,13 +281,47 @@ export const getMergedTxHistory = createSelector(
             tx.from,
             network.id
           );
+          const derivedTxType = deriveTxType(txTypeMetas, accounts, tx)
+          const valueTransfers = tx.valueTransfers || []
+          if (valueTransfers.length == 0 && !bigify(tx.value).isZero()) {
+            valueTransfers.push({
+              asset: tx.baseAsset,
+              to: tx.to,
+              from: tx.from,
+              amount: fromTokenBase(bigify(tx.value), DEFAULT_ASSET_DECIMAL).toString(),
+              isNFTTransfer: false
+            } as IFullTxHistoryValueTransfer);
+          }
+
+          // handles unknown internal transaction value transfer in exchange tx types.
+          // @todo: remove when we have access to internal transactions
+          const accountsMap = accounts.reduce((acc, cur) => {
+            acc[cur.address.toLowerCase()] = true
+            return acc;
+          }, {} as {[key: string]: boolean });
+          if (
+            txTypeMetas[derivedTxType]
+            && txTypeMetas[derivedTxType].type == 'EXCHANGE'
+            && (valueTransfers.filter((t) => accountsMap[t.to.toLowerCase()]) || []).length == 0
```

Comment:
it's looking for a "received" value transfer in a transaction that has txType of action type "EXCHANGE". Some exchanges (uniswap as an example) will have internal transactions to handle swapping from token -> ETH. We don't have internal transactions handling in backend yet, but we still want to show the user that they received something in exchange for the token they sent to uniswap, instead of just leaving the "received" column empty - so I added a generic unknown base asset value transfer.